modules/boost.config/1.88.0/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+module(
+    name = "boost.config",
+    version = "1.88.0",
+    bazel_compatibility = [">=7.2.1"],
```

Comment:
bazel version must be >= 7.6.0 at least, see https://github.com/bazelbuild/bazel/releases/tag/7.6.0

Also for 8.x, it should be >= 8.2.0, but there is no good way to express both at the same time I think... 

@Wyverald

Reply:
changed 7.2.0 to 7.6.0